api/src/main/java/com/gentics/mesh/etc/config/HttpServerConfig.java

```diff
@@ -23,15 +28,21 @@
 
 	public static final int DEFAULT_HTTP_PORT = 8080;
 
+	public static final int DEFAULT_HTTPS_PORT = 8443;
+
 	public static final String DEFAULT_CERT_PATH = "config/cert.pem";
 	public static final String DEFAULT_KEY_PATH = "config/key.pem";
+	public static final ClientAuth DEFAULT_CLIENT_AUTH_MODE = ClientAuth.REQUEST;
```

Comment:
Are the SSL handshakes  more concise when this is set to `ClientAuth.NONE`? If so maybe rethink the default value.